test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
Yet another possibility for readable-and-an-improvement-over-what's-there-now is to leave it as strings but remove the inline concatenation by removing the `CRLF` variable:

```js
  'HTTP/1.1 200 OK\r\n' +
  'Content-types: text/plain\r\n' +
  'Content-Length: 4\r\n' +
  '\r\n' +
  'pong'
```

Aside: Is `Content-types` is a typographical error and it is supposed to be `Content-type`? If so, maybe that should be fixed too.